app/controllers/ExclusionListController.scala

```diff
@@ -147,13 +149,11 @@ class ExclusionListController @Inject()(
 
   def withOrWithoutNinoDecision(isCurrentTaxYear: String, iabdType: String): Action[AnyContent] =
     (authenticate andThen noSessionCheck).async { implicit request =>
-      val iabdTypeValue = uriInformation.iabdValueURLDeMapper(iabdType)
       if (exclusionsAllowed) {
-        val taxYearRange = controllersReferenceData.YEAR_RANGE
         val resultFuture = formMappings.binaryRadioButton
           .bindFromRequest()
           .fold(
-            formWithErrors =>
+            _ =>
               Future(
                 Redirect(routes.ExclusionListController.withOrWithoutNinoOnPageLoad(isCurrentTaxYear, iabdType))
                   .flashing("error" -> Messages("ExclusionDecision.noselection.error"))),
```

Comment:
flashing the error would cause the error page to be removed if the user swapped language, displaying the error from the form would be better

Reply:
sorry it would be the same either, momentarily forgot how the toggle workds, but by flashing the error message instead of displaying it by form prevents us from using some aspects of play ui

Reply:
Yeah, there's a few instances of this in the service - I didn't want to go too out of scope and re-write everything so left as much of it the same as possible. I think in this case the formWithErrors was never actually used so I just swapped it to an "_". Happy to look into fixing this though

Reply:
With the nature of the change I think this would be a good time to update it and I don't really see it being hugely out of scope, we can check with the rest of the team with what they think about this.

Reply:
Ok, have sorted this - now using formWithErrors to show errors rather than flashing. I think there's some more flashing elsewhere in the service so I'll see what can be done there too